scp.c

```diff
@@ -2077,10 +2077,14 @@ sink(int argc, char **argv, const char *src)
 		omode = mode;
 		mode |= S_IWUSR;
 #ifdef WINDOWS
-		if (add_mark_of_web(np) == -1 && verbose_mode) {
+		wchar_t* filepath = utf8_to_utf16(np);
```

Comment:
Maybe have two errors for each error condition?  It might be easier to debug.

```c
if (filepath == NULL) {
    run_error("scp: %s: failed to add mark of the web. Cannot convert file path to utf16\n", np);
    exit(1);
}
if (add_mark_of_web(filepath) == -1) {
    free(filepath);
    run_err("scp: %s: failed to add mark of the web\n", np);
    exit(1);
}
free(filepath);
```